### PR TITLE
Fix Reexecute

### PR DESCRIPTION
**Pull request by kabir-vats**
… all workflows

### CODE REVIEW

**Review by pagrubel — COMMENTED** [review]
This looks great and worked well for me. 
@kabir-vats What is the file that has the name of the workflow that appears in the archive? 
total 5
-rw-rw-r-- 1 pagrubel pagrubel  912 Sep  4 15:16 bee.conf
-rw-rw-r-- 1 pagrubel pagrubel    3 Sep  4 15:16 bee_wf_name
-rw-rw-r-- 1 pagrubel pagrubel    8 Sep  4 15:16 bee_wf_status
drwxrwxr-x 2 pagrubel pagrubel 4096 Sep  4 15:16 bee_workflow
drwxrwxr-x 2 pagrubel pagrubel 4096 Sep  4 15:16 cat-174318f74bf24b6bab36a1c4f6068519
-rw-rw-r-- 1 pagrubel pagrubel 2159 Sep  4 15:16 cat-grep-tar
drwxrwxr-x 2 pagrubel pagrubel 4096 Sep  4 15:16 grep0-8dd6f1b3a2074a1280561718f31f7c06
drwxrwxr-x 2 pagrubel pagrubel 4096 Sep  4 15:16 grep1-e4fa3ccb9fbf4cf492ca69505725d2e3
-rw-rw-r-- 1 pagrubel pagrubel  211 Sep  4 15:16 submit_command_args.yaml
drwxrwxr-x 2 pagrubel pagrubel 4096 Sep  4 15:16 tar-83bb47ee861d4d0cb0fbb5cf1f836f19

**Comment by pagrubel** [comment]
The submit command I used was:
beeflow submit cgt cat-grep-tar cat-grep-tar/workflow.cwl cat-grep-tar/input.yml .

so I'm asking about the cat-grep-tar file

**Comment by kabir-vats** [comment]
> The submit command I used was: beeflow submit cgt cat-grep-tar cat-grep-tar/workflow.cwl cat-grep-tar/input.yml .
> 
> so I'm asking about the cat-grep-tar file

If you mean the file that holds all the cwl files, that should be bee_workflow.

**Comment by pagrubel** [comment]
> > The submit command I used was: beeflow submit cgt cat-grep-tar cat-grep-tar/workflow.cwl cat-grep-tar/input.yml .
> > so I'm asking about the cat-grep-tar file
> 
> If you mean the file that holds all the cwl files, that should be bee_workflow.

No there is an extra binary file with the name of the workflow, in this case cat-grep-tar. It doesn't appear when running using the current develop branch.

File: beeflow/wf_manager/resources/wf_list.py [hunk 1]
```diff
@@ -121,7 +117,7 @@ def post(self):
         return make_response(jsonify(msg='Workflow uploaded', status='ok',
                              wf_id=wf_id), 201)
 
-    def put(self):
+    def put(self):  # This method can be deleted / deprecated
```
Comment (pagrubel) [inline beeflow/wf_manager/resources/wf_list.py:120]: Why not delete it now?

**Comment by pagrubel** [comment]
@kabir-vats This looks great. Thank you for streamlining code.

**Review by pagrubel — APPROVED** [review]
This looks very good Kabir, thank you.

**Pull request merged**